src/ddmd/semantic.d

```diff
@@ -54,7 +54,7 @@ extern(C++) Initializer semantic(Initializer init, Scope* sc, Type t, NeedInterp
 }
 
 // Performs semantic analysis in Statement AST nodes
-extern(C++) Statement semantic(Statement s, Scope* sc)
+extern(C++) Statement xxsemantic(Statement s, Scope* sc)
```

Comment:
I guess now that you've found and replaced all of the call-sites this function can now be removed.

Reply:
If I remove it now, then the other PRs will have merge conflicts, and I gotta do a rebase. Simpler to just do it later.